### ISSUE TITLE
Podman 1.9 Release?

### ISSUE THREAD

**Issue by leonsmith** [boot2podman/boot2podman#21]
I would love to get invovled in helping roll a new image for the 1.9 release but I feel this is a little out of my depth.

Only asking as if you install the latest podman-client and podman-machine (which uses  boot2podman as its image) you end up with the following versions missmatched version.

```
Version:            1.9.4-dev
RemoteAPI Version:  1
Go Version:         go1.13.10
Git Commit:         67f236701cee2d19b37e203356652a769d974c2f
Built:              Fri May 22 21:11:18 2020
OS/Arch:            darwin/amd64

Server:
Version:            1.7.1-dev
RemoteAPI Version:  1
Go Version:         go1.12.10
Git Commit:         ce4bf33ccc80819f199bdbb6526b9adee1ad41b0-dirty
Built:              Sat Jan 18 10:01:17 2020
OS/Arch:            linux/amd64
```

I'm not actaully sure if having a lower server version is bad for the client and what the backwards compability guarentees are.

**Comment by afbjorklund** [comment]
We had a lot of issues with 1.9.0 and 1.9.1, but it seems like 1.9.3 is better.

There is a pending upgrade of the OS, and should get some 1.8/1.9 out...

Thanks for the reminder!

**Comment by afbjorklund** [comment]
Now we even have a 2.0 to strive for, looking to see if I should include REST

**Comment by afbjorklund** [comment]
Most likely will not do anything for 2.0 and REST, upstream wants to handle

**Comment by afbjorklund** [comment]
Fixed in v0.27
